kgl-jogl/src/main/kotlin/com/danielgergely/kgl/KglJogl.kt

```diff
@@ -144,29 +144,43 @@ class KglJogl(private val gl: GL) : Kgl {
 
     override fun uniform1f(location: UniformLocation, f: Float) = gl.glUniform1f(location, f)
 
+    override fun uniform1fv(location: UniformLocation, value: FloatArray) = gl.glUniform1fv(location, value.size, value, 0)
+
     override fun uniform1i(location: UniformLocation, i: Int) = gl.glUniform1i(location, i)
 
+    override fun uniform1iv(location: UniformLocation, value: IntArray) = gl.glUniform1iv(location, value.size, value, 0)
+
     override fun uniform2f(location: UniformLocation, x: Float, y: Float) = gl.glUniform2f(location, x, y)
 
+    override fun uniform2fv(location: UniformLocation, value: FloatArray) = gl.glUniform2fv(location, value.size, value, 0)
+
     override fun uniform2i(location: UniformLocation, x: Int, y: Int) = gl.glUniform2i(location, x, y)
 
+    override fun uniform2iv(location: UniformLocation, value: IntArray) = gl.glUniform2iv(location, value.size, value, 0)
+
     override fun uniform3f(location: UniformLocation, x: Float, y: Float, z: Float) = gl.glUniform3f(location, x, y, z)
 
-    override fun uniform3fv(location: UniformLocation, value: FloatArray) = gl.glUniform3fv(location, 1, value, 0)
```

Comment:
This doesn't look right. Based on my understanding, `count` is supposed to be the number of typed elements to assign, not the number of primitives. For example, in `uniform3fv`, if the `count` is 2, then OpenGL is about to update 2 instances of vec3f (6 floats in total), and not just two floats.

I believe the best options we can consider with `uniform*v` calls is to:
A) Expose the `count` parameter through Kgl functions
B) Hide the `count` parameter in Kgl, and calculate it automatically. For example, for `uniform3fv` it should be `value.size / 3`

It seems that [WebGL1](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/uniform) doesn't support the `count` parameterer, so with WebGL1 only the **B)** option is possible, however [WebGL2](https://developer.mozilla.org/en-US/docs/Web/API/WebGL2RenderingContext/uniform) `uniform*v` calls do have the `srcOffset` and `srcLength` parameters, which makes option **A)** possible

So moving forward, regarding to this PR I'd suggest going with option **B)** (both for existing and new *fv functions) as it offers WebGL1 compatibility, and option **A)** can be implemented later at any time

Reply:
Thanks @gergelydaniel, make the change you describe in B), thanks for catching that!